doc/source/myrocks/zenfs.rst

```diff
@@ -11,101 +11,191 @@ which places files into zones on a raw zoned block device (ZBD) using
+      $ sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo percona-release setup ps80
 
-    zenfs df --zbd=nvme1n2
-    Free: 7563 MB
-    Used: 0 MB
-    Reclaimable: 0 MB
-    Space amplification: 0%
+2. Install the `zenfs` package. The Percona Server for MySQL with MyRocks and the ZenFS plugin package is listed in the :ref:`installing_from_binary_tarball` section of the *Percona Server for MySQL* installation instructions. 
+   
+   .. code-block:: bash
+	
+      $ sudo apt install percona-server-server
+	
+#. Install the RocksDB plugin package. This package copies ``ha_rocksdb.so`` into a predefined location. **The RocksDB storage engine is not enabled**.
+
+   .. code-block:: bash
+	
+	  $ sudo apt install percona-server-rocksdb
+
+
+.. _zenfs-configure
+
+Configuration
+============================================================
+
+#. Identify your ZBD device, ``<block_device_name>``, with ``lsblk``. Add the ``-o`` option to select which columns are returned. 
+
+   In the example, the ``NAME`` column returns the block device name, the ``SIZE`` column returns the size of the device, and the ``ZONED`` column returns information if the device uses the zone model. The value, ``host-managed``, identifies a ZBD model.
+
+   .. sourcecode:: bash
+
+      lsblk -o NAME,SIZE,ZONED
+      NAME        SIZE  ZONED
+      sda       247.9G  none
+      |-sda1    230.9G  none
+      |-sda2        1G  none
+      |-sda3       16G  none
+      sdb        15.5T  host-managed
+
+		
+#. Change the ownership of ``nvme0n2`` to the ``mysql:mysql`` user account.
+
+   .. code-block:: bash
+	
+	  $ sudo chown mysql:mysql /dev/nvme0n2
+		
+#. Change the permissions so that the user or owner can read and write and the MySQL group can read, in case they must take a backup, for ``nvme0n2``.
+
+   .. code-block:: bash
+	
+	  $ sudo chmod 640 /dev/nvme0n2
+
+#. Change the scheduler for ``nvme0n2`` to ``mq_deadline``.
+
+   .. code-block:: bash
+	
+	  $ sudo bash -c 'echo mq-deadline > /sys/class/block/nvme0n2/queue/scheduler'
+
+#. Create an auxiliary directory for ZenFS, for example, ``/var/lib/mysql_aux``. The ZenFS auxiliary directory is a regular (POSIX) file directory used internally to resolve file locks and shared access. There are no strict requirements for the location but the directory must be write accessible for the `mysql:mysql` UNIX system user account. Each ZBD must have an individual auxiliary directory. This directory is recommended to be at the same level as "/var/lib/mysql", which is the default Percona Server for MySQL directory. 
+
+#. Initialize ZenFS on ``nvme0n2``.
+
+   .. code-block:: bash
+	
+	  $ sudo -H -u mysql zenfs mkfs --zbd=nvme0n2 --aux_path=/var/lib/mysql_zenfs_aux_ nvme0n2 --finish_threshold=0 --force
+		
+   .. note::
+	
+		 If you must configure ZenFS to use a directory inside ``/var/lib`` (owned by ``root:root`` without write permissions for other user accounts), do the following steps manually:
```

Comment:
Won't SELinux/AppArmor prevent the process from writing to these arbitrary locations? Do we take this into account?

Reply:
Yes, @LenzGr you are right. Most probably `/var/lib/mysql_zenfs_aux_<zbd_name>` will be blocked by SELinux/AppArmor. This requires further investigation. At the moment I suggest to add a note
```
You may need to manually update AppArmor/SELinux rules to give mysqld write access to
/var/lib/mysql_zenfs_aux_<zbd_name> 
```

Reply:
Updated based on discussion.